.github/workflows/update-flake-lock.yml

```
@@ -20,3 +20,12 @@ jobs:
           pr-labels: |
             dependencies
             automated
+      - name: Enable auto-merge
+        if: ${{ success() }}
+        run: |
+          gh pr merge --auto --rebase "$PR_URL"
+        env:
+          PR_URL: ${{ steps.update-flake-lock.outputs.pull-request-url }}
```

actionlint (1.7.7)

property "update-flake-lock" is not defined in object type {}

(expression)